app/api/server/v1/voip/extensions.ts

```diff
@@ -4,22 +4,22 @@ import { Match, check } from 'meteor/check';
 import { API } from '../../api';
 import { Voip } from '../../../../../server/sdk';
 import { IVoipExtensionBase } from '../../../../../definition/IVoipExtension';
-import { IVoipConnectorResult } from '../../../../../definition/IVoipConnectorResult';
+// import { IVoipConnectorResult } from '../../../../../definition/IVoipConnectorResult';
 
 // Get the connector version and type
 API.v1.addRoute('connector.getVersion', { authRequired: true }, {
-	get() {
-		const version = Promise.await(Voip.getConnectorVersion());
+	async get() {
```

Comment:
I tried making this function async and remove Promise.await. But I did not receive the expected results. It would be good to check if we get the desired data.  My knowledge is limited about the API class. Nevertheless, I suspect that it may not be calling get with await.

Reply:
@amolghode1981 do you remember if your attempt was before this PR: https://github.com/RocketChat/Rocket.Chat/pull/23701 ?
we introduce this feature there, so before was not possible tu use async functions on endpoints

Reply:
It wasn't before, but the livechat-voip branch wasn't updated when Amol tried to use async handlers 👀